### PR TITLE
Add DTEAM's peer, rest API endpoint, RPC endpoint to chain4energy chain.json

### PR DESCRIPTION
**Pull request by doodleoo3**

### CODE REVIEW

File: chain4energy/chain.json [hunk 1]
```diff
@@ -294,6 +294,11 @@
       }
     ],
     "persistent_peers": [
+      {
+        "id": "00eb92db69a4848f8ac1c84e7b8b2d9b01b95dff",
+        "address": "peer.c4e.mainnet.dteam.tech:30656",
+        "provider": "DTEAM"
+      }
```
Comment (reversesigh) [inline chain4energy/chain.json:301]: Add a comma here

Comment (doodleoo3) [reply chain4energy/chain.json]: done

File: chain4energy/chain.json [hunk 2]
```diff
@@ -678,6 +695,11 @@
       "url": "https://explorer.apeironnodes.com/chain4energy",
       "tx_page": "https://explorer.apeironnodes.com/chain4energy/transactions/${txHash}"
     },
+    {
+      "kind": "explorer",
```
Comment (JeremyParish69) [inline chain4energy/chain.json:699]: I think kind here should be the host (DTEAM)

Comment (doodleoo3) [reply chain4energy/chain.json]: yep

just changed it, thanks

**Pull request merged**